src/DataTable/tests/DataTable.test.jsx

```diff
@@ -101,8 +113,48 @@ describe('<DataTable />', () => {
     const wrapper = mount(<DataTable {...props} />);
     expect(wrapper.find(TableFooter).length).toEqual(1);
   });
-  // TODO: test that useTable is called with the correct arguments
+  it('adds a column when table is selectable', () => {
+    const wrapper = mount(<DataTable {...props} isSelectable />);
+    const tableHeaders = wrapper.find(Table).find('th');
+    expect(tableHeaders.length).toEqual(props.columns.length + 1);
+  });
+  it('adds additional columns', () => {
+    const wrapper = mount(<DataTable {...props} additionalColumns={additionalColumns} />);
+    const tableHeaders = wrapper.find(Table).find('th');
+    expect(tableHeaders.length).toEqual(props.columns.length + additionalColumns.length);
+    expect(wrapper.text()).toContain(additionalColumns[0].Header);
+    expect(wrapper.text()).toContain(additionalColumns[1].Header);
+  });
+  test('calls useTable with the data and columns', () => {
+    const spy = jest.spyOn(reactTable, 'useTable');
+    mount(<DataTable {...props} />);
+    expect(spy).toHaveBeenCalledTimes(1);
+    expect(spy.mock.calls[0][0].columns).toEqual(props.columns);
+    expect(spy.mock.calls[0][0].data).toEqual(props.data);
+    expect(spy.mock.calls[0][0].initialState).toEqual({});
+    expect(spy.mock.calls[0]).toHaveLength(2);
+  });
+  test.each([
+    [{}, { manualFilters: false, manualPagination: false, manualSortBy: false }],
+    [{ manualFilters: true }, { manualFilters: true, manualPagination: false, manualSortBy: false }],
+    // [{ manualPagination: true }, { manualFilters: true, manualPagination: true, manualSortBy: false }],
+    // [{ manualSortBy: true }, { manualFilters: true, manualPagination: true, manualSortBy: true }],
```

Comment:
Why are these here?

Reply:
Oh, because I was debugging the tests and didn't want to run them all. I'll turn them back on.